src/main/java/com/shanebeestudios/skbee/elements/other/events/EvtSpawnerSpawn.java

```diff
@@ -0,0 +1,62 @@
+package com.shanebeestudios.skbee.elements.other.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.Getter;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.SpawnerSpawnEvent;
+import org.jetbrains.annotations.Nullable;
+
+public class EvtSpawnerSpawn extends SkriptEvent {
+
+    static {
+        Skript.registerEvent("Spawner - Spawn", EvtSpawnerSpawn.class, SpawnerSpawnEvent.class,
+                        "spawner spawn[ing] [of %entitydatas%]6")
```

Comment:
Im assuming the "6" was by mistake?
```suggestion
                        "spawner spawn[ing] [of %entitydatas%]")
```